galleries/examples/ticks/date_formatters_locators.py

```diff
@@ -55,13 +34,42 @@ def plot_axis(ax, locator=None, xmax='2002-02-01', fmt=None, formatter=None):
     ax.text(0.0, 0.2, locator or formatter, transform=ax.transAxes,
             fontsize=14, fontname='Monospace', color='tab:blue')
 
+# %%
+# :ref:`Date locators <date-tickers>` (also known as tickers)
```

Comment:
I added this comment because the example calls them `locators` but they are called `tickers` in the linked docs. Not sure what's the _"official"_ name.

Reply:
So the docs should read "Most of the date ~tickers~ **locators** can locate single or multiple ~values~ **ticks.**" and "The available date ~tickers~ **locators** are:", right? If this is the case, should this be changed in this PR or better in a seperate one (or do I still get it wrong?).  
So I will remove the "(also known as tickers)".

Reply:
let's make it a separate issue